modules/imaging_browser/test/imaging_browser_test_plan.md

```diff
@@ -11,7 +11,7 @@
 
 ### B. ViewSession / Volume List
 
-8. Sidebar:  all links work 
+8. Sidebar:  all links work, inlcuding the Download DICOM option
```

Comment:
- typo: inlcuding -> including
- If there are multiple tars, they should be listed (and download on each should work)

Reply:
@christinerogers 
1) typo fixed! thanks
2) yes, this should work; I tested it as we had a case in CCNA wwhich had its initial MRi visit split into two scanning sessions and the changes here work as well. I fixed the screenshot at the beginning of the pull request to show this case.